packages/dev/test/common/ContractInteractor.test.ts

```diff
@@ -85,14 +92,22 @@ contract('ContractInteractor', function (accounts) {
           gasPrice: '1',
           pctRelayFee: '0',
           baseRelayFee: '0',
+          transactionCalldataGasUsed: '0',
           relayWorker: workerAddress,
           forwarder: constants.ZERO_ADDRESS,
           paymaster: pm.address,
           paymasterData: '0x',
           clientId: '1'
         }
       }
-      const ret = await contractInteractor.validateRelayCall(200000, relayRequest, '0x', '0x')
+      const blockGasLimit = await contractInteractor._getBlockGasLimit()
+      const encodedData: RelayCallABI = {
+        maxAcceptanceBudget: '200000',
+        relayRequest,
+        signature: '0x',
+        approvalData: '0x'
+      }
+      const ret = await contractInteractor.validateRelayCall(encodedData, new BN(blockGasLimit))
```

Comment:
should we pass the blockGasLimit (from CI) into the CI.validateRelayCall ?

Reply:
This is a test. In a real code, a configurable value is used by the `validateRelayCall` and this PR makes it no longer fall back to the block gas limit.